packages/combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -352,10 +340,6 @@ export const ComboBoxMixin = (subclass) =>
         input.setAttribute('autocorrect', 'off');
 
         this._revertInputValueToValue();
-
-        if (this.clearElement) {
```

Comment:
This logic was placed in this observer because of `vaadin-combo-box-light` timings, moved it to `ready()`.